search/includes/classes/class-search.php

```diff
@@ -1482,6 +1489,26 @@ public function filter__ep_prepare_meta_allowed_protected_keys( $keys, $post ) {
 		return \apply_filters( 'vip_search_post_meta_allow_list', $keys, $post );
 	}
 
+	/**
+	 * Since we've established that enabling the protected content feature causes attachments
+	 * to be indexed, we should ensure that 'attachment' is in the indexable post types if
+	 * protected content is enabled.
+	 *
+	 * @param array $indexable_post_types Current list indexable post types in VIP Search.
+	 * @return array New list of indexable post types in VIP Search.
+	 */
+	public function filter__ep_indexable_post_types( $indexable_post_types ) {
```

Comment:
What do you think about naming this method according to content? Or is that some code style to name following the filter?

I would suggest `filter__ep_indexable_post_types__add_attachment_to_post_types` or something like that. I know this is super long name, but maybe better descriptive?

Reply:
Otherwise, it looks great!

Reply:
Switch it. We're not bound by convention 😆